src/types/formatted/qaGroup.ts

```diff
@@ -0,0 +1,15 @@
+import { Wysiwyg as FormattedWysiwyg } from './wysiwyg'
+
+type QaGroupQa = {
+  title: string
+  answer: FormattedWysiwyg
+}
+
+export type QaGroup = {
```

Comment:
`PublishedParagraph & {...}`

Reply:
We'll need to add all of these paragraph types to the `Paragraph` component in `src/templates/components/paragraph/index.tsx`